apps/cmd/pusher/main.go

```diff
@@ -8,14 +8,16 @@ import (
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/pkgerrors"
 	"github.com/spf13/cobra"
+
+	"github.com/Stork-Oracle/stork_external/lib/pusher"
 )
 
 var verbose bool
 
 func main() {
 	rootCmd := &cobra.Command{
-		Use:   "stork",
-		Short: "Stork CLI tool for EVM contract updates based on WebSocket prices",
+		Use:   "stork-push",
```

Comment:
how do you feel about calling this app something like `chain_pusher` and making this command `stork-chain-pusher`? Don't feel super strongly about it, but `stork-push` is a little general - sounds like it could be pushing into stork too.